### PR TITLE
Add dependabot config

### PR DESCRIPTION
**Pull request by distantnative**

### CODE REVIEW

File: .github/dependabot.yml  [hunk 1]
```diff
@@ -0,0 +1,21 @@
+version: 1
```
Comment (lukasbestle) [inline .github/dependabot.yml ]:
```suggestion
version: 2
```

File: .github/dependabot.yml  [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+updates:
+  - package-ecosystem: "composer"
+    directory: "/"
```
Comment (lukasbestle) [inline .github/dependabot.yml :4]: To commit the vendor files:

```suggestion
  - package-ecosystem: "composer"
    directory: "/"
    vendor: true
```

**Review by lukasbestle — DISMISSED** [review]

**Comment by neildaniels** [comment]
This is awesome!

I would recommend increasing the open PR limit to something higher than the default of 5—especially if this is only going to run at a weekly cadence.

https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#open-pull-requests-limit

**Comment by distantnative** [comment]
@neildaniels good thinking - didn't catch that default, especially since they spell five out :D

**Comment by afbora** [comment]
> error: invalid path '.github/dependabot.yml '

@distantnative `dependabot.yml` filename ending with space and I can't fetch the develop branch. Could you fix that?

https://github.com/getkirby/kirby/blob/develop/.github/dependabot.yml%20

**Comment by distantnative** [comment]
@afbora done 🙏

**Pull request merged**